subprojects/plugins/src/main/java/org/gradle/api/tasks/GroovyRuntime.java

```diff
@@ -58,7 +61,10 @@
 public class GroovyRuntime {
     private static final VersionNumber GROOVY_VERSION_WITH_SEPARATE_ANT = VersionNumber.parse("2.0");
     private static final VersionNumber GROOVY_VERSION_REQUIRING_TEMPLATES = VersionNumber.parse("2.5");
-    private static final VersionNumber GROOVY_VERSION_REQUIRING_JSONXML = VersionNumber.parse("3.0");
+
+    private static final List<String> GROOVY3_LIBS = Arrays.asList(
+        "groovy", "groovy-ant", "groovy-templates", "groovy-json", "groovy-xml", "groovy-groovydoc");
```

Comment:
We should consider adding all modules that we decided to pull in by default given we dropped `-all`. That would be

- `groovy`
- `groovy-ant`
- `groovy-astbuilder`
- `groovy-console`
- `groovy-datetime`
- `groovy-dateutil`
- `groovy-groovydoc`
- `groovy-json`
- `groovy-nio`
- `groovy-sql`
- `groovy-templates`
- `groovy-test`
- `groovy-xml`

See also https://docs.gradle.org/7.0-rc-1/userguide/upgrading_version_6.html#groovy_modularization